cluster-operator/src/main/java/io/strimzi/operator/cluster/model/EntityOperator.java

```diff
@@ -102,7 +101,7 @@ public String getZookeeperConnect() {
     }
 
     public static String secretName(String cluster) {
-        return cluster + CERTS_SUFFIX;
+        return KafkaResources.entityOperatorDeploymentName(cluster) + CERTS_SUFFIX;
```

Comment:
Shouldn't we declare a new method on `KafkaResources` to do this: `entityOperatorCertificatesSecretName(String cluster)`? And presumably we should have a call site for that in place where we are trying to use the secret (which is why this breakage happened in the first place)?